src/screens/ContactsScreen/ContactsScreen.tsx

```diff
@@ -0,0 +1,33 @@
+import {useCallback} from 'react';
+import {Linking, Text, View} from 'react-native';
+import { StyledButton } from 'components';
+import { useThemedStyles } from 'styles/commonStyles';
+import styles from './ContactsScreenStyles';
+
+export default function ContactsScreen() {
+  const themedStyles = useThemedStyles();
+  
+  const handleEmailPress = useCallback(() => {
```

Comment:
when you see that callback are doing the same - nice to combine them in one - and extend with some props
for example here you can pass a param like communication - which can be `type Communication = 'sms' | 'phone' | 'email'` - and just build correct url based on it